data/json/recipes/tools/tool.json

```
@@ -1591,5 +1591,20 @@
     "qualities": [ { "id": "GRIND", "level": 2 } ],
     "tools": [ [ [ "hotcut", -1 ] ] ],
     "components": [ [ [ "scrap_aluminum", 2 ] ], [ [ "plastic_chunk", 1 ] ] ]
+  },
+  {
+    "result": "makeshift_welding_blanket",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "tailor",
+    "difficulty": 2,
+    "time": "120 m",
+    "autolearn": [ [ "tailor", 3 ] ],
+    "book_learn": [ [ "welding_book", 1 ] ],
+    "using": [ [ "sewing_aramids", 50 ], [ "adhesive", 4 ] ],
+    "proficiencies": [ { "proficiency": "prof_polymerworking", "required": false, "time_multiplier": 1.5 } ],
+    "components": [ [ [ "rock_wool_bat", 15 ] ] ]
```

```suggestion
    "components": [ [ [ "rock_wool_bat", 22 ] ] ]
```

This at least matches weights with some extra for scrap ends.